README.md

```diff
@@ -94,9 +93,13 @@ using you local (git-ignored) `.env` file as a canonical source.
 
 ### Multiple Buildpacks
 
-Multiple buildpack support used to be unofficial and relied on [a custom buildpack created by David Dollar](https://github.com/ddollar/heroku-buildpack-multi.git). This is no longer the case since Heroku has rolled out official support for multiple buildpacks.
+Multiple buildpack support used to be unofficial and relied on [a custom buildpack created
+by David Dollar](https://github.com/ddollar/heroku-buildpack-multi.git). This is no longer
+the case since Heroku has rolled out official support for multiple buildpacks.
 
-Therefore, if you decide to deploy Orientation on Heroku manually (without using the Heroku button, which would take care of this for you) you will need to add two buildpacks since the app relies on NodeJS for yarn package installation.
+Therefore, if you decide to deploy Orientation on Heroku manually (without using the Heroku button,
+which would take care of this for you) you will need to add two buildpacks since the app relies
+on NodeJS for Bower package installation.
```

Comment:
This is wrong, we don't use Bower. Can you restore the previous copy? Thanks.

Reply:
Right, looks like the new setup uses yarn/npm to install assets and webpacker for building.

Reply:
Reverted that change to the README